src/module_wrap.cc

```diff
@@ -47,12 +45,17 @@ ModuleWrap::ModuleWrap(Environment* env,
 }
 
 ModuleWrap::~ModuleWrap() {
-  Local<Module> module = module_.Get(Isolate::GetCurrent());
-  std::vector<ModuleWrap*>* same_hash = module_map_[module->GetIdentityHash()];
-  auto it = std::find(same_hash->begin(), same_hash->end(), this);
+  Isolate* isolate = Isolate::GetCurrent();
+  Environment* env = Environment::GetCurrent(isolate);
```

Comment:
`ModuleWrap` is a `BaseObject`, right?  You can just use `env()` here.  Same goes for the constructor: `env()->isolate()`.  There are probably quite a few more places that could be updated.

Reply:
Yes, done.